README.md

```diff
@@ -197,45 +204,3 @@ If you want to cite our 🤗 Datasets [paper](https://arxiv.org/abs/2109.02846)
     primaryClass={cs.CL},
 }
 ```
-```bibtex
-@software{quentin_lhoest_2021_5639822,
```

Comment:
I removed this @albertvillanova since researchers can just cite the paper, WDYT ?

Reply:
I added a link to Zenodo with the list of all the released DOIs sorted by version

Reply:
Good idea! we should give both options for citation: either the library, or a specific version of our software package.